src/resume_parser/templates/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Resume Keyword Extractor</title>
+    <link rel="stylesheet" href="/static/style.css">
+</head>
+<body>
+    <div class="container">
+        <h1>Resume Keyword Extractor</h1>
+        <form id="uploadForm" enctype="multipart/form-data">
+            <input type="file" name="file" id="fileInput" accept=".pdf,.docx" required>
+            <button type="submit">Upload</button>
+        </form>
+        <div id="keywords"></div>
+    </div>
+    <script>
+        document.getElementById('uploadForm').onsubmit = async (e) => {
+            e.preventDefault();
+            const formData = new FormData();
+            formData.append('file', document.getElementById('fileInput').files[0]);
+
+            const response = await fetch('/upload', {
```

Comment:
There's no error handling in the fetch request. If the server is down, the file isn't properly uploaded, or the response is invalid, it will silently fail. Consider adding error handling using a try-catch block around the fetch request and checking response.ok to handle server errors gracefully.

Reply:
Resolved by stuti in the next pull request